cirq/iqm/iqm_device_test.py

```diff
@@ -0,0 +1,58 @@
+import cirq
+import pytest
+
+
+def test_valid_operations():
+    adonis = cirq.iqm.Adonis()
+
+    adonis.validate_operation(cirq.GateOperation(cirq.X,
```

Comment:
Are all angles supported for this device or just pi pulses?  I would add a test for cirq.XPowGate(exponent=0.25) or similar.

Reply:
All angles are supported, good idea to add a test for other angles as well 👍🏼

Reply:
Added tests in https://github.com/quantumlib/Cirq/pull/2701/commits/9d67ae0d39c1695320dc8ecc5596daacf345db03